### PR TITLE
TVIST1-484: Added hearing post attachments

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/TVIST1-484

Adds ordered attachments to hearing posts.

### CODE REVIEW

File: src/Controller/HearingController.php [hunk 1]
```diff
@@ -195,7 +251,28 @@ public function hearingPostForward(CaseEntity $case, HearingPost $hearingPost):
...
+        $digitalPost->setDocument($hearingPost->getDocument());
+        $digitalPost->setEntityType(get_class($case));
+        $digitalPost->setEntityId($case->getId());
+
+        // Handle attachments
+        $attachments = $hearingPost->getAttachments();
+
+        foreach ($attachments as $attachment) {
+            $digitalPostAttachment = new DigitalPostAttachment();
+            $digitalPostAttachment->setPosition($attachment->getPosition());
```
Comment (rimi-itk) [inline src/Controller/HearingController.php:265]: This is handled by `DigitalPost` (cf. https://github.com/itk-dev/naevnssekretariatet/blob/develop/src/Entity/DigitalPost.php#L230-L238)

File: src/Controller/HearingController.php [hunk 2]
```diff
@@ -195,7 +251,28 @@ public function hearingPostForward(CaseEntity $case, HearingPost $hearingPost):
...
+            $digitalPostAttachment->setDocument($attachment->getDocument());
+            $digitalPostAttachment->setDigitalPost($digitalPost);
```
Comment (rimi-itk) [inline src/Controller/HearingController.php:267]: This is handled by `DigitalPost` (cf. https://github.com/itk-dev/naevnssekretariatet/blob/develop/src/Entity/DigitalPost.php#L208)

File: src/Controller/HearingController.php (hunk 1 above)
Comment (jekuaitk) [reply src/Controller/HearingController.php]: Fixed in a3d58663c826d21bd47a98dee75f3b1ba9610973.

File: src/Controller/HearingController.php (hunk 2 above)
Comment (jekuaitk) [reply src/Controller/HearingController.php]: Fixed in a3d58663c826d21bd47a98dee75f3b1ba9610973.

File: src/Controller/HearingController.php [hunk 3]
```diff
@@ -122,7 +131,37 @@ public function hearingPostCreate(CaseEntity $case, DocumentRepository $document
...
+
+            // Create new file from template
+            $fileName = $mailTemplateHelper->renderMailTemplate($hearingPost->getTemplate(), $case);
+
+            // Compute fitting name
+            $documentUploader->specifyDirectory('/case_documents/');
+            $updatedFileName = $slugger->slug($hearingPost->getTemplate()->getName()).'-'.uniqid().'.pdf';
+
+            // Rename file
+            $filesystem->rename($fileName, $documentUploader->getDirectory().'/'.$updatedFileName, true);
```
[the remark this answers is not in the document]
Comment (jekuaitk) [reply src/Controller/HearingController.php]: Moved to `DocumentUploader` in ed7738b8288442bcf3ffb1be905ed07ff596d9e8.

**Review by jekuaitk — APPROVED** [review]

**Pull request merged**